src/Nancy/Json/Json.cs

```diff
@@ -118,10 +118,11 @@ public static bool IsJsonContentType(string contentType)
             var contentMimeType = contentType.Split(';')[0];
 
             return contentMimeType.Equals("application/json", StringComparison.OrdinalIgnoreCase) ||
-                   contentMimeType.StartsWith("application/json-", StringComparison.OrdinalIgnoreCase) ||
-                   contentMimeType.Equals("text/json", StringComparison.OrdinalIgnoreCase) ||
-                  (contentMimeType.StartsWith("application/vnd", StringComparison.OrdinalIgnoreCase) &&
-                   contentMimeType.EndsWith("+json", StringComparison.OrdinalIgnoreCase));
+            contentMimeType.StartsWith("application/json-", StringComparison.OrdinalIgnoreCase) ||
+            contentMimeType.Equals("text/json", StringComparison.OrdinalIgnoreCase) ||
+            (contentMimeType.StartsWith("application/vnd", StringComparison.OrdinalIgnoreCase) &&
+            contentMimeType.EndsWith("+json", StringComparison.OrdinalIgnoreCase)) ||
+            contentMimeType.Equals("application/merge-patch+json");

```

Comment:
@grumpydev It's `StartsWith("application/vnd") && EndsWith("+json")`.

@jchannon Yeah. Let's just check `EndsWith("+json")`. I think that should be enough. Remember to update the XML doc :wink: